File: Euclid.Storage/Database/Data/Catalogue/CatalogueData.cs
using FluentNHibernate.Mapping;
using System;

namespace Euclid.Storage.Database.Data
{
    class CatalogueMapping : ClassMap<CatalogueData>
    {
        public CatalogueMapping()
        {
            Table("catalogue");
            Id(x => x.Id, "id");
            Map(x => x.Section, "section");
            Map(x => x.Price, "price");
            Map(x => x.DefinitionId, "definition_id");
        }
    }

    public class CatalogueData
    {
        public virtual string Id { get; set; }
        public virtual string Section { get; set; }
        public virtual int Price { get; set; }
        public virtual int DefinitionId { get; set; }
    }

}

File: euclid-storage/src/main/java/euclid/storage/database/data/CatalogueData.java
package euclid.storage.database.data;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "catalogue")
public class CatalogueData {

    @Id
    @Column(name = "id")
    private String id;

    @Column(name = "section")
    private String section;

    @Column(name = "price")
    private int price;

    @Column(name = "definition_id")
    private int definitionId;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getSection() {
        return section;
    }

    public void setSection(String section) {
        this.section = section;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getDefinitionId() {
        return definitionId;
    }

    public void setDefinitionId(int definitionId) {
        this.definitionId = definitionId;
    }
}
